### PR TITLE
AddingDocumentsPage

### PR DESCRIPTION
**Pull request by AlexKaschuk**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by MykhailoKlapchuk — APPROVED** [review]

File: EPlast/EPlast.BLL/DTO/MethodicDocument/MethodicDocumentWraperDTO.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+using System.Collections.Generic;
+using System.Text;
+
+namespace EPlast.BLL.DTO
+{
+   public class MethodicDocumentWraperDTO
+    {
+        public MethodicDocumentDTO MethodicDocument { get; set; }
+        public string FileAsBase64 { get; set; }
+
```
Comment (LanchevychMaxym) [inline EPlast/EPlast.BLL/DTO/MethodicDocument/MethodicDocumentWraperDTO.cs:11]: line

**Review by LanchevychMaxym — APPROVED** [review]
Good job!

**Review by tetyanakutsko — APPROVED** [review]
Great 👍

File: EPlast/EPlast.BLL/Services/MethodicDocument/MethodicDocumentService.cs [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+            {
+                MethodicDocument = new MethodicDocumentDTO()
+            };
+        }
+
+        public async Task DeleteMethodicDocumentAsync(int id)
+        {
+            var document = (await _repoWrapper.MethodicDocument.GetFirstAsync(d => d.ID == id));
+            if (document == null)
+                throw new ArgumentNullException($"Decision with {id} id not found");
```
Comment (voroniak) [inline EPlast/EPlast.BLL/Services/MethodicDocument/MethodicDocumentService.cs:69]: Decision?

**Pull request merged**